datafusion/core/tests/sqllogictests/src/insert/mod.rs

```diff
@@ -74,12 +72,20 @@ pub async fn insert(ctx: &SessionContext, insert_stmt: &SQLStatement) -> Result<
             .collect::<std::result::Result<Vec<DFExpr>, DataFusionError>>()?;
         // Directly use `select` to get `RecordBatch`
         let dataframe = ctx.read_empty()?;
-        insert_batches.push(dataframe.select(logical_exprs)?.collect().await?)
+        insert_batches.extend(dataframe.select(logical_exprs)?.collect().await?)
```

Comment:
I wonder if you could make this

```suggestion
        origin_batches.extend(dataframe.select(logical_exprs)?.collect().await?)
```

And avoid the need for `insert_batches` entirely 🤔

Reply:
Yeah, nice suggestion!